region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -479,15 +493,26 @@
         }
       },
       "requires": [
+        {"or": [
+          "canWalljump",
+          "HiJump",
```

Comment:
HiJump and the other options aside from `canWalljump` are redundant, since `canPreciseWalljump` is assumed in every case anyway?